dist_tables/dml.rst

```diff
@@ -3,7 +3,13 @@
 Ingesting, Modifying Data (DML)
 ###############################
 
-The following code snippets use the distributed tables example dataset, see :ref:`ddl`.
+The following code snippets use the distributed tables example, see :ref:`ddl`.
+We also use the github events dataset to illustrate the commands below. You can download that dataset by running:
```

Comment:
It looks like the bulk loading section is the only place that uses this dataset. What about moving the wget/gzip code block down there like

```rst
...you can directly use PostgreSQL’s \\COPY command. First download our example dataset:

.. code-block:: bash

  wget http://examples.citusdata.com/github_archive/github_events-2015-01-01-{0..5}.csv.gz
  gzip -d github_events-2015-01-01-*.gz

Then copy the data using psql:

.. code-block:: postgresql

  \COPY github_events FROM 'github_events-2015-01-01-0.csv' WITH (format CSV)
```